client/httplib/httplib.go

```diff
@@ -131,6 +139,11 @@ func (b *BeegoHTTPRequest) GetRequest() *http.Request {
 	return b.req
 }
 
+// Context returns the request context
+func (b *BeegoHTTPRequest) Context() context.Context {
```

Comment:
I think we don't need this function. I can not image that why users want to get the context.